controller/internal/routes/base_router.go

```diff
@@ -192,9 +192,9 @@ func ListWithEnvelopeFactory(rc *response.RequestContext, toEnvelope ApiListEnve
 
 	meta := &rest_model.Meta{
 		Pagination: &rest_model.Pagination{
```

Comment:
Did you have to switch these to pointers to get it to work, or was a stylistic preference?

Reply:
I altered the swagger document to set the pagination fields as 'required' which `go-swagger` interprets as needing to be a pointer to check for nil vs zero value.